package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Template plugin for Fastify",
   "main": "index.js",
   "scripts": {
-    "test": "standard && tap test.js"
+    "example": "node example.js",
+    "example-with-options": "node example-ejs-with-some-options.js",
+    "test": "standard && cross-env TAP_SNAPSHOT=1 tap test.js"
```

Comment:
I think the normal test run should not have TAP_SNAPSHOT=1 set.

Reply:
uh sorry but so I don't know how to do the change, could you give me some hint/help ? sorry ...

Reply:
You need two tasks, one `npm run create-snaphost`  with `TAP_SNAPSHOT=1`, and the normal `npm test` without `TAP_SNAPSHOT`.

`TAP_SNAPSHOT=1` creates/updates the snapshot

Reply:
ah ok thanks ... but is it good the same to run the same tests in both or do you prefer to split in another test source (this option could be too much) ?

Reply:
Yes.